src/msca/metrics/main.py

```diff
@@ -0,0 +1,475 @@
+from typing import Optional, List, Union
+from enum import StrEnum
+import numpy as np
+import pandas as pd
+
+from sklearn import metrics
+
+class ErrorMetric(StrEnum):
```

Comment:
Since we want to merge `skill` into this function, let's call this class `Metric`. Sorry about the back and forth.